.travis.yml

```diff
@@ -0,0 +1,19 @@
+dist: trusty
+sudo: false
+
+language: node_js
+node_js:
+  - '10'
+
+before_install:
+  - export CHROME_BIN=chromium-browser
+
+cache:
+  directories:
+    - ./node_modules
+
+install:
+  - npm install
+
+script:
+  - npm run test -- --watch=false --no-progress --browsers=ChromeHeadlessNoSandbox
```

Comment:
you don't have any tests :)

Reply:
Yes it is needed to build successfully on Travis , without it the app fails on Travis. We tried removing it and searched on it. [check this out](https://medium.com/faun/configuring-travis-ci-for-angular-application-34afee1715f)